frontend/src/components/ParticipantInfo.js

```diff
@@ -104,26 +107,32 @@ const ParticipantInfo = observer(() => {
   const [programList, setProgramList] = React.useState([])
   // list of all services
   const [serviceList, setServiceList] = React.useState([])
+  // ???
```

Comment:
Unhelpful comment

Reply:
should be // TODO: refactor honestly, next push issue